app/views/vehicles/new.html.erb

```diff
@@ -0,0 +1,36 @@
+<%= form_with model: [:user, @vehicle], local: true do |form| %>
+
+  <div class="field">
+    <%= form.label :make %>
+    <%= form.text_field :make, required: true %>
+  </div>
+
+  <div class="field">
+    <%= form.label :model %>
+    <%= form.text_field :model, required: true %>
+  </div>
+
+  <div class="field">
+    <%= form.label :year %>
+    <%= form.text_field :year, required: true %>
+  </div>
+
+  <div class="field">
+    <%= form.label :awd %>
+    <%= form.check_box :awd %>
+  </div>
+
+  <div class="field">
+    <%= form.label :storage_rack %>
+    <%= form.check_box :storage_rack %>
+  </div>
+
+  <div class="field">
+    <%= form.label :total_seats %>
+    <%= form.number_field :total_seats, min: 1, required: true %>
+  </div>
```

Comment:
Repeating myself, but I just thought that we might want to be super clear when having drivers fill this out that this includes the driver (I think...?)